sklearn/utils/validation.py

```diff
@@ -675,10 +686,14 @@ def check_X_y(X, y, accept_sparse=False, accept_large_sparse=True,
         it is converted to float64. Should only be used for regression
         algorithms.
 
-    warn_on_dtype : boolean (default=False)
+    warn_on_dtype : boolean or None, optional (default=None)
         Raise DataConversionWarning if the dtype of the input data structure
         does not match the requested dtype, causing a memory copy.
 
+        ..deprecated:: 0.21
```

Comment:
```suggestion
        .. deprecated:: 0.21
```

Needs a space to be rendered correctly. 

Also you don't need the quotes surrounding the message here.

(Please fix the other one as well).

Reply:
done